Modules/Map/QuestieMap.lua

```diff
@@ -332,6 +332,26 @@ function QuestieMap.ProcessQueue()
     end
 end
 
+local weaponMasterSkills = {
```

Comment:
As you already said: This new logic should be placed somewhere else. QuestieMap seems the wrong place for it. Maybe a small new module is fine.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5568/commits/e21f912fd598b3550179a024653b37afa5335831